.github/postgresql.yaml

```diff
@@ -0,0 +1,6 @@
+primary:
+  persistence:
+    enabled: false
+
+auth:
+  postgresPassword: postgres
```

Comment:
This is a configuration file for [PostgreSQL packaged by Bitnami](https://github.com/bitnami/charts/tree/main/bitnami/postgresql).